File: Lab2Scripts/LAB2JAVA/Direccion.java
public class Direccion {
    private String calle;
    private String nomenclatura;
    private String barrio;
    private String ciudad;
    private String edificio;
    private String apto;

    public Direccion() {}

    // Setters
    public void setCalle(String calle) {
        this.calle = calle;
    }

    public void setNomenclatura(String nomenclatura) {
        this.nomenclatura = nomenclatura;
    }

    public void setBarrio(String barrio) {
        this.barrio = barrio;
    }

    public void setCiudad(String ciudad) {
        this.ciudad = ciudad;
    }

    public void setEdificio(String edificio) {
        this.edificio = edificio;
    }

    public void setApto(String apto) {
        this.apto = apto;
    }

    // Getters
    public String getCalle() {
        return calle;
    }

    public String getNomenclatura() {
        return nomenclatura;
    }

    public String getBarrio() {
        return barrio;
    }

    public String getCiudad() {
        return ciudad;
    }

    public String getEdificio() {
        return edificio;
    }

    public String getApto() {
        return apto;
    }

    // Método toString
    public String toString() {
        return calle + " " + nomenclatura + ", " + barrio + ", " + ciudad + ", Edificio " + edificio + ", Apto " + apto;
    }
}

File: Lab2Scripts/LAB2PYTHON/direccion.py
class Direccion:
    def __init__(self):
        self._calle = ""
        self._nomenclatura = ""
        self._barrio = ""
        self._ciudad = ""
        self._edificio = ""
        self._apto = ""

    def set_calle(self, calle):
        self._calle = calle

    def set_nomenclatura(self, nomenclatura):
        self._nomenclatura = nomenclatura

    def set_barrio(self, barrio):
        self._barrio = barrio

    def set_ciudad(self, ciudad):
        self._ciudad = ciudad

    def set_edificio(self, edificio):
        self._edificio = edificio

    def set_apto(self, apto):
        self._apto = apto

    def get_calle(self):
        return self._calle

    def get_nomenclatura(self):
        return self._nomenclatura

    def get_barrio(self):
        return self._barrio

    def get_ciudad(self):
        return self._ciudad

    def get_edificio(self):
        return self._edificio

    def get_apto(self):
        return self._apto

    def __str__(self):
        return f"{self._calle} {self._nomenclatura}, {self._barrio}, {self._ciudad}, Edificio {self._edificio}, Apto {self._apto}"
